projects/packages/sync/src/class-defaults.php

```diff
@@ -189,6 +189,7 @@ class Defaults {
 		'wpcom_reader_views_enabled',
 		'wpcom_site_setup',
 		'wpcom_subscription_emails_use_excerpt',
+		'jetpack_verbum_subscription_modal',
```

Comment:
From a Sync perspective, is there a WPCOM counterpart diff that adds this option in the Sync options' whitelist on WPCOM?
Also just a double check that you actually need this setting synced :)

Reply:
Thanks for noticing this. 
I can confirm we need this setting synced as we will use this on Jetpack and WPCOM. I'll create the WPCOM diff

Reply:
Diff D129916-code